src/Home/Navbar.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styles from "./style.module.css";
+import { links } from "../constants";
+
+const Navbar = () => {
+  return (
+    <nav className={styles.nav}>
+      <div>
+        <a href="/" aria-label="Logo">
+          <img src="/random.svg" alt="Logo" />
+        </a>
+      </div>
+      <ul className={styles.navLinks}>
+        {links.map((link) => (
```

Comment:
what if `links` here is undefined, this mapping will fail and result in a WSOD. What can we do here, to prevent it?